app/models/collection_draft.rb

```diff
@@ -97,11 +97,11 @@ def update_draft(params, editing_user_id)
 
       if new_draft
         self.draft = new_draft
-        save
+        return save
       elsif self.draft != {}
         # draft had content, but now new_draft is nil/empty so any data has been deleted and saved in the forms
         self.draft = {}
-        save
+        return save
```

Comment:
This function would save, then regardless of the outcome return true (below).  These mean that the response from save is provided back to the calling function for evaluation.